### PR TITLE
Restrict asset upload

### PR DESCRIPTION
**Pull request by Kartones**
Closes https://github.com/CartoDB/cartodb-management/issues/4289

### CODE REVIEW

**Comment by Kartones** [comment]
@rafatower CR please

File: app/models/asset.rb [hunk 1]
```diff
@@ -6,7 +6,9 @@ class Asset < Sequel::Model
 
   KIND_ORG_AVATAR = 'orgavatar'
 
-  PUBLIC_ATTRIBUTES = %W{ id public_url user_id kind }
+  PUBLIC_ATTRIBUTES = %w{ id public_url user_id kind }
+
+  VALID_EXTENSIONS = %w{ jpg jpeg gif png svg }
```
Comment (rafatower) [inline app/models/asset.rb]: who can confirm this is the complete list of assets valid types?

Comment (Kartones) [inline app/models/asset.rb]: @xavijam , spoke with him but just in case summoning him

**Comment by rafatower** [comment]
:+1:

File: app/models/asset.rb (hunk 1 above)
Comment (xavijam) [inline app/models/asset.rb]: IMHO, those are the only types we should allow to upload.

Comment (Kartones) [inline app/models/asset.rb]: Yep, this are the ones on the UI literal listing allowed images. BMP and TIFF are too big in general for markers or BGs, and we don't support here compressed files so... I also think it is more than enough

**Pull request merged**